app.js

```diff
@@ -0,0 +1,117 @@
+    response.status(200).send("<h1 style='font-style: italic; color: #ff0066; position: absolute; top: 50%; left: 10%; width: 400px; height: 300px; margin-top: -150px;'>D'Oh</h1>")
+  })
+
+app.get('/JJ-Evans', (request, response) => {
+    response.status(200).send("<h1 style='font-style: italic; color: #ff0066; position: absolute; top: 50%; left: 10%; width: 400px; height: 300px; margin-top: -150px;'>Dy-no-myte!</h1>")
+  })
+  
+app.get('/Batman', (request, response) => {
+    response.status(200).send("<h1 style='font-style: italic; color: #ff0066; position: absolute; top: 50%; left: 10%; width: 400px; height: 300px; margin-top: -150px;'>To the Batmobile!</h1>")
+  })
+
+app.get('/Hannibal-Smith', (request, response) => {
+    response.status(200).send('<h1 style="font-style: italic; color: #ff0066; position: absolute; top: 50%; left: 10%; width: 400px; height: 300px; margin-top: -150px;">I love it when a plan comes together</h1>')
+  })
+
+app.get('/Jack-Dawson', (request, response) => {
+    response.status(200).send('<h1 style="font-style: italic; color: #ff0066; position: absolute; top: 50%; left: 10%; width: 400px; height: 300px; margin-top: -150px;">I\'m king of the world!</h1>')
+  })
+
+app.get('/Frankenstein', (request, response) => {
+    response.status(200).send('<h1 style="font-style: italic; color: #ff0066; position: absolute; top: 50%; left: 10%; width: 400px; height: 300px; margin-top: -150px;">It\'s alive! It\'s alive</h1>')
+  })
+
+app.get('/Tony-Montana', (request, response) => {
+    response.status(200).send('<h1 style="font-style: italic; color: #ff0066; position: absolute; top: 50%; left: 10%; width: 400px; height: 300px; margin-top: -150px;">Say hello to my little friend</h1>')
+  })
+
+app.get('/Dorothy', (request, response) => {
+    response.status(200).send('<h1 style="font-style: italic; color: #ff0066; position: absolute; top: 50%; left: 10%; width: 400px; height: 300px; margin-top: -150px;">Toto, I\'ve got a feeling we\'re not in Kansas anymore</h1>')
+  })
+
+app.get('/Rod-Tidwell', (request, response) => {
+    response.status(200).send('<h1 style="font-style: italic; color: #ff0066; position: absolute; top: 50%; left: 10%; width: 400px; height: 300px; margin-top: -150px;">Show me the money!</h1>')
+  })
+
+  const magic8Responses = [
+    "It is certain",
+    "It is decidedly so",
+    "Without a doubt",
+    "Yes - Definitely",
+    "You may rely on it",
+    "As I see it, yes",
+    "Most likely",
+    "Outlook good",
+    "Yes, and signs point to yes",
+    "Reply hazy, try again",
+    "Ask again later",
+    "Better not tell you now",
+    "Cannot predict now",
+    "Concentrate and ask again",
+    "Don't count on it",
+    "My reply is no",
+    "My sources say no",
+    "Outlook not so good",
+    "Very doubtful"
+  
+  ]
+  let catchPhrase = ''
+  let result = ''
+
+app.get('/magic8', (request, response) => {
+    for (let i = 0; i < magic8Responses.length; i++){
```

Comment:
I think the loop is unnecessary. You just need line 101 which will get a random response that you can send back.